recipes/humap/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "humap" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 19fb2764fabb46b1f701afefe38ca7c2297737fcd11428e6dfbc937fda6aa65c
+  
+build:
+  script:
+    - python setup.py build_ext -I$PREFIX/include/eigen3 install  # [osx]
+    - python setup.py build_ext -I$PREFIX/include/eigen3 install  # [linux]  
+    - python setup.py build_ext -I%PREFIX%\Library\include\eigen3 install  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - libgomp  # [linux]
+    - numpy >=1.23.0
```

You need to pin numpy here to a version. Otherwise, the newest one will be used. 
```suggestion
    - numpy 1.23
```

---

Done, @xhochy. All checks have passed!